### PR TITLE
Update LICENSE

### PR DESCRIPTION
**Pull request by gnought**

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -1,22 +1,21 @@
-Copyright (c) 2015-2019 Matteo Collina, http://matteocollina.com
```
Comment (mcollina) [inline LICENSE]: That's not how this works. Please retain this line and place it after the new one.

File: LICENSE [hunk 2]
```diff
@@ -1,22 +1,21 @@
...
 
-Permission is hereby granted, free of charge, to any person
-obtaining a copy of this software and associated documentation
-files (the "Software"), to deal in the Software without
-restriction, including without limitation the rights to use,
-copy, modify, merge, publish, distribute, sublicense, and/or sell
-copies of the Software, and to permit persons to whom the
-Software is furnished to do so, subject to the following
-conditions:
+Copyright (c) 2020 moscajs
```
Comment (mcollina) [inline LICENSE]:
```suggestion
Copyright (c) Aedes Contributors
```

File: LICENSE (hunk 1 above)
Comment (gnought) [reply LICENSE]: Thanks for pointing out. :)

**Comment by gnought** [comment]
Should I follow the same practice for other aedes project LICENSE?

**Comment by mcollina** [comment]
yes please
Il 31 gen 2020, 10:45 +0100, Gnought <notifications@github.com>, ha scritto:

**Pull request merged**